stealth.lua

```diff
@@ -0,0 +1,107 @@
+
+cloaking.privs_to_string = minetest.privs_to_string
+function minetest.privs_to_string(privs, delim)
+    assert(type(privs) == "table")
+    delim = delim or ","
+    local list = {}
+    for priv, bool in pairs(privs) do
+        if bool and priv ~= "cloaking" then
+            list[#list + 1] = priv
+        end
+    end
+    return table.concat(list, delim)
```

Comment:
```lua
table.concat(list, delim or ",")
```

Reply:
I already have `delim = delim or ","` above; do you think it'd be more appropriate here?

Reply:
I was meaning removing that line and adding `delim or ","` there, it shouldn't affect anything though.